README.md

```diff
@@ -472,7 +472,7 @@ docker run --rm -it \
   -e STORAGE=local \
   -e STORAGE_LOCAL_ROOTDIR=/charts \
   -v $(pwd)/charts:/charts \
-  ghcr.io/helm/chartmuseum:v0.16.1
+  ghcr.io/helm/chartmuseum:canary
```

Comment:
can we add some explanation here that the canary is our main version after the action is done , most people asked it in the issue list . I'd prefer keep the tag here. And also we should update www .

Reply:
I reverted to use the released version. eventually we'll want to remove/automate this to make releases easier